vexim/config/header.php

```diff
@@ -105,4 +114,6 @@
   }
   if (isset($_GET['login']) && ($_GET['login'] == "failed")) { print _("Login failed"); }
 
+  if ($validateAsSiteadmin) print '<a href="site.php" style="float:right; margin-right:20px;">siteadmin home</a>';
```

Comment:
Can you create a new class in the style.css? I started to remove inline css-code to be able to use content security policy properly.